lsif/src/server/backend/database.ts

```diff
@@ -130,7 +124,8 @@ export class Database {
     }
 
     /**
-     * Return a list of locations which reference the symbol at the given position.
+     * Return a list of locations that reference the symbol at the given position. The
+     * resulting list of locations does not contain duplicates.
```

Comment:
```suggestion
     * Return a list of unique locations that reference the symbol at the given position.
```

or maybe "distinct"?